node/prisma/README.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+1. Node/Npm/Npx
+2. YugabyteDB 
+
+# Build and run
+
+Create Database
```

Comment:
Add a note that, by default it tries to connect to a cluster on localhost. To change it, edit `.env` file accordingly.